entrypoint.sh

```diff
@@ -91,6 +91,11 @@ fi
 if [[ $KUBERNETES ]]; then
     # enable kubernetes check
     cp /etc/dd-agent/conf.d/kubernetes.yaml.example /etc/dd-agent/conf.d/kubernetes.yaml
+
+    # enable event collector
+    if [[ $KUBERNETES_COLLECT_EVENTS ]]; then
+        sed -i -e "s@# collect_events: False@collect_events: True@" /etc/dd-agent/conf.d/kubernetes.yaml
```

Comment:
Oh... I actually used the convention I found on the config file, specifically `use_histogram: False`